tests/features/joinup_group/short_id.feature

```diff
@@ -0,0 +1,130 @@
+@api @terms
+Feature:
+  As an owner of a group
+  In order to make the URL of my group easier to read
+  I need to be able to provide a short ID to my group.
+
+  Scenario: Short IDs are unique in collections.
+    Given the following collections:
+      | title                     | short ID | state     |
+      | Scientific Studies        | SSt-2020 | validated |
+      | Scientific Sustainability | SST-2020 | validated |
```

Comment:
This is making 2 collections with the same case-insensitive short IDs, shouldn't this be impossible, even on an API level?